diff --git a/sample/systemd/borgmatic.service b/sample/systemd/borgmatic.service
index 3ec9710e..a975ceb4 100644
--- a/sample/systemd/borgmatic.service
+++ b/sample/systemd/borgmatic.service
@@ -2,9 +2,6 @@
 Description=borgmatic backup
 Wants=network-online.target
 After=network-online.target
-# Prevent borgmatic from running unless the machine is plugged into power. Remove this line if you
-# want to allow borgmatic to run anytime.
-ConditionACPower=true
 
 [Service]
 Type=oneshot

